pkg/apis/wksprovider/controller/manifests/manifests_vfsdata.go

```diff
@@ -21,42 +21,42 @@ var Manifests = func() http.FileSystem {
 	fs := vfsgen۰FS{
 		"/": &vfsgen۰DirInfo{
 			name:    "/",
-			modTime: time.Date(2020, 5, 19, 22, 23, 27, 249616053, time.UTC),
+			modTime: time.Date(2020, 6, 7, 19, 44, 6, 833861756, time.UTC),
```

Comment:
These changes are all spurious - just the last-modified time changes and not the content.

Reply:
Same story as https://github.com/weaveworks/wksctl/pull/209#discussion_r441516514 should I ignore this when pushing?

Reply:
Reverted